packages/node-cli/src/commands/AddDependenciesCommandGenerator.js

```diff
@@ -152,15 +169,16 @@ module.exports = class AddDependenciesCommandGenerator extends BaseCommandGenera
 		return dependenciesString;
 	}
 
-	_formatOutput(operationResult) {
-		if (SDKOperationResultUtils.hasErrors(operationResult)) {
-			SDKOperationResultUtils.logResultMessage(operationResult);
-			SDKOperationResultUtils.logErrors(operationResult);
+	_formatOutput(actionResult) {
+		if (actionResult.error) {
+			if (actionResult.resultMessage) {
```

Comment:
Why do we read resultMessage if the status is error? I think all errors should be in error messages in this case